bulk_site_creator/schema.py

```diff
@@ -102,6 +103,7 @@ def __init__(
         self.section = section
         self.department_id = department_id
         self.course_group_id = course_group_id
+        self.sis_account_id = sis_account_id
```

Comment:
sis_account_id will also need to be added to the to_dict method.
We can also remove sis_account_id from the Job schema since each task record will have that field.